tripal/src/TripalVocabTerms/PluginManagers/TripalCollectionPluginManager.php

```diff
@@ -35,15 +35,15 @@ public function __construct(
       ,CacheBackendInterface $cache_backend
       ,ModuleHandlerInterface $module_handler
       ,$pluginInterface
-      ,$pluginDefinitionAnnotation
+      ,$pluginDefinitionAttribute
```

Comment:
docblock and variables need to match.

```suggestion
      ,$plugin_definition_attribute_name
```

Reply:
Renamed with commit https://github.com/tripal/tripal/pull/2239/commits/af24a63b4966cd47dd1bffeee49cc64270e69776